### PR TITLE
Landmarks wrapper and gaze_estimation_demo

### PR DESCRIPTION
**Pull request by VoronovaIntern**

### CODE REVIEW

**Comment by VoronovaIntern** [comment]
@ygnn123  in post-processing of the facial-landmarks-98-detection-0001 model, the points are drawn with bias to the right, the reason for this is not clear, if you know what the problem is, tell me.

**Comment by anzhella-pankratova** [comment]
Jenkins ok to test

**Comment by vladimir-dudnik** [comment]
13:08:21  + ./venv/bin/python ./src/ci/check-basics.py
13:08:21  running miscellaneous checks...
13:08:22  demos/common/cpp/models/include/models/results.h:113: trailing whitespace.
13:08:22  + 
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:67: trailing whitespace.
13:08:22  +    
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:72: trailing whitespace.
13:08:22  +    
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:78: trailing whitespace.
13:08:22  +        cv::resize(img, resizedImage, scaledSize, 0, 0, cv::INTER_CUBIC); 
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:96: trailing whitespace.
13:08:22  +    
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:130: trailing whitespace.
13:08:22  +    
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:154: trailing whitespace.
13:08:22  +        cv::Mat coord(3, 1, CV_32F); 
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:182: trailing whitespace.
13:08:22  +        
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:197: trailing whitespace.
13:08:22  +       
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:200: trailing whitespace.
13:08:22  +       
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:209: trailing whitespace.
13:08:22  +    return 0; 
13:08:22  demos/common/cpp/models/src/landmarks_model.cpp:247: new blank line at EOF.
13:08:22  demos/gaze_estimation_demo/cpp/main.cpp:183: trailing whitespace.

**Comment by VoronovaIntern** [comment]
Jenkins please retry a build

**Comment by anzhella-pankratova** [comment]
Jenkins please retry a build

**Comment by VoronovaIntern** [comment]
No matching models: "facial-landmarks-98-detection-0001" in tests

**Comment by vladimir-dudnik** [comment]
@VoronovaIntern new model will be uploaded later, for now you may comment out test case for it

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/gaze_estimation_demo/cpp/main.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2018 Intel Corporation
+﻿// Copyright (C) 2018 Intel Corporation
```
Comment (vladimir-dudnik) [inline demos/gaze_estimation_demo/cpp/main.cpp:1]:
```suggestion
﻿// Copyright (C) 2018-2021 Intel Corporation
```

File: demos/gaze_estimation_demo/cpp/main.cpp [hunk 2]
```diff
@@ -94,12 +97,16 @@ int main(int argc, char *argv[]) {
         // Set up face detector and estimators
         FaceDetector faceDetector(ie, FLAGS_m_fd, FLAGS_d_fd, FLAGS_t, FLAGS_fd_reshape);
         HeadPoseEstimator headPoseEstimator(ie, FLAGS_m_hp, FLAGS_d_hp);
-        LandmarksEstimator landmarksEstimator(ie, FLAGS_m_lm, FLAGS_d_lm);
+       // LandmarksEstimator landmarksEstimator(ie, FLAGS_m_lm, FLAGS_d_lm);
```
Comment (vladimir-dudnik) [inline demos/gaze_estimation_demo/cpp/main.cpp:100]: why line is commented?

**Pull request closed without merging**